test/SpecRunner.html

```diff
@@ -24,6 +25,7 @@
 
   <!-- include source files here... -->
   <script src="../../fast-json-patch/src/json-patch-duplex.js"></script>
+  <script src="../../../node_modules/jsonpatcherproxy/dist/jsonpatcherproxy.min.js"></script>
   <script src="../src/palindrom.js"></script>
   <script src="../src/palindrom-dom.js"></script>
 
```

Comment:
Something is not right here, it looks to me like we should either:

1. Load `fast-json-patch` from `node_modules`
2. Or, load `palindrom-dom.js` bundle from dist

Maybe it is not worth refactoring anyway, because we are migrating to Mocha soon.